episodes/backwards-design.md

```diff
@@ -1,101 +1,83 @@
+exercises: 30
 ---
 
 :::::::::::::::::::::::::::::::::::::: questions 
 
-- How do you write a lesson using R Markdown and `{sandpaper}`?
+- What is meant by _backwards design_ of curriculum?
+- What are the main steps of the backwards design process?
+- What are the advantages and disadvantages of backwards design?
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ::::::::::::::::::::::::::::::::::::: objectives
 
-- Explain how to use markdown with the new lesson template
-- Demonstrate how to include pieces of code, figures, and nested challenge blocks
+By the end of this session, participants should be able to...
 
-::::::::::::::::::::::::::::::::::::::::::::::::
-
-## Introduction
-
-This is a lesson created via The Carpentries Workbench. It is written in
-[Pandoc-flavored Markdown][pandoc] for static files (with extension `.md`) and
-[R Markdown][r-markdown] for dynamic files that can render code into output
-(with extension `.Rmd`). Please refer to the [Introduction to The Carpentries
-Workbench][carpentries-workbench] for full documentation.
-
-What you need to know is that there are three sections required for a valid
-Carpentries lesson template:
-
- 1. `questions` are displayed at the beginning of the episode to prime the
-    learner for the content.
- 2. `objectives` are the learning objectives for an episode displayed with
-    the questions.
- 3. `keypoints` are displayed at the end of the episode to reinforce the
-    objectives.
-
-:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+- discuss the application of backwards design to curriculum for short-format training events
+  like Carpentries workshops.
+- list at least three advantages and at least one disadvantage of 
+  taking a backwards design approach to lesson/curriculum design.
 
-Inline instructor notes can help inform instructors of timing challenges
-associated with the lessons. They appear in the "Instructor View"
-
-::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
-
-::::::::::::::::::::::::::::::::::::: challenge 
-
-## Challenge 1: Can you do it?
-
-What is the output of this command?
+::::::::::::::::::::::::::::::::::::::::::::::::
 
-```r
-paste("This", "new", "lesson", "looks", "good")
-```
+:::::::::::::::::::::::::::::::::::::::::::::::::: checklist
 
-:::::::::::::::::::::::: solution 
+## Reading List
 
-## Output
- 
-```output
-[1] "This new lesson looks good"
-```
+- FIXME
 
-:::::::::::::::::::::::::::::::::
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
 
-## Challenge 2: how do you nest solutions within challenge blocks?
+:::::::::::::::::::::::::::::::::::::::::::::::::: discussion
 
-:::::::::::::::::::::::: solution 
+## Introductions
 
-You can add a line with at least three colons and a `solution` tag.
+The first discussion session will begin with a full round of Introductions,
+where each trainee will be asked to share some details about themselves
```

Comment:
folks leading this session should probably introduce themselves as well?

Reply:
```suggestion
where trainees and facilitators will be given the opportunity to share some details about themselves
```